Gemfile.lock

```diff
@@ -238,6 +242,7 @@ DEPENDENCIES
   pg (~> 1.1)
   puma (~> 5.0)
   rails (~> 7.0.4.1)
+  rails-controller-testing
```

Comment:
- [ ] As requested by the previous reviewer Kindly make sure you have and you are useing RSpec gem for testing.  you only wrote testing without adding the gem. Kindly go to the previous reviewers comment for more information

Reply:
@Munsa1 thanks so much; can you explain your requested changes? Could you give me a clue to this please; I didn't get it well.